float32 roll
float32 pitch
float32 yaw
float32 depth
float32 pos_x
float32 pox_y
float32 pos_z
